src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/state/AssessmentStateMachine.kt

```diff
@@ -65,6 +65,7 @@ val assessmentStateMachine = StateMachine.create<AssessmentState, AssessmentLife
   }
 
   state<AwaitingAddressAndRiskChecks> {
+    on<SubmitForAddressChecks> { dontTransition() }
```

Comment:
this was the bug fix.. umming and arring as to whether it is a bug or not (page changes may make it irrelevant) but think it makes sense to fix for now

Reply:
Nice and simple

Reply:
Would be nice to ave a integration test for this

Reply:
I added a unit test to cover this